src/app/components/elements/EventBookingForm.tsx

```diff
@@ -14,6 +14,112 @@ interface EventBookingFormProps {
   updateAdditionalInformation: (update: AdditionalInformation) => void;
 }
 
+const SchoolYearGroup = ({
+  event,
+  additionalInformation,
+  updateAdditionalInformation,
+}: {
+  event: AugmentedEvent;
+  additionalInformation: AdditionalInformation;
+  updateAdditionalInformation: (update: AdditionalInformation) => void;
+}) => {
+  return (
+    <>
+      <RS.Label htmlFor="year-group" className="form-required">
+        School year group
+      </RS.Label>
+      <RS.Input
+        type="select"
+        id="year-group"
+        name="year-group"
+        value={additionalInformation.yearGroup || ""}
+        onChange={(event) => updateAdditionalInformation({ yearGroup: event.target.value })}
+      >
+        <option value="" />
+        {event.isAStudentEvent && (
+          <>
+            <option value="9">Year 9</option>
+            <option value="10">Year 10</option>
+            <option value="11">Year 11</option>
+            <option value="12">Year 12</option>
+            <option value="13">Year 13</option>
+          </>
+        )}
+        {!event.isStudentOnly && (
+          <>
+            <option value="TEACHER">N/A - Teacher</option>
+            <option value="OTHER">N/A - Other</option>
+          </>
+        )}
+      </RS.Input>
+      {event.isStudentOnly && (
+        <div className="text-muted" data-testid="student-only">
```

Comment:
I have noticed that you tend to use `data-testid` quite often. While that by itself is not a problem, I wanted to mention that this method is the least recommended one, it's the last option when nothing else makes more sense. Here's the recommended priority list, with an explanation:
https://testing-library.com/docs/queries/about/#priority

You don't need to change anything in this test, but please bear this in mind when writing future tests. If you're feeling the need to use `data-testid`, always check if there is another more appropriate query that you could be using instead.

Reply:
Yeah I try to opt for getByRole or by Label where possible (neither of which would work well for a div). I generally prefer to avoid getByText for looking for a conditionally rendered section like this example simply because if the section does render but the text content changes, the test won't find it, even though it might technically be operating correctly if you see what I mean.

Reply:
Yeah, that makes sense. As long as you've considered other options before using `data-testid`, then it's all good :-)